core/server/src/main/java/tachyon/master/TachyonMaster.java

```diff
@@ -134,6 +134,28 @@ public static void main(String[] args) {
   private long mStartTimeMs = -1;
 
   /**
+   * @return a list of the enabled master services' names
+   */
+  public static List<String> getNames() {
+    List<String> names = Lists.newArrayList();
```

Comment:
As a work around, I cached the service loader and service names in TachyonMaster.